### PR TITLE
Open repo

### LINKED ISSUE #3: Open repo on github with fantastic-kit
I hate that I need to manually open browser and find my repo on github
Should be trivial

### PR DESCRIPTION
**Pull request by Azhng**
Closes #3 

```
fk repo
```

### CODE REVIEW

File: bin/fk.sh [hunk 1]
```diff
@@ -52,6 +58,16 @@ fpr()
...
 }
 
+frepo()
+{
+  if ! git rev-parse --git-dir &>/dev/null; then
+    echo "not in a git repo"
+    return 1
+  fi
+  read repoName ownerName < <(remote-info)
+  xdg-open https://github.com/$ownerName/$repoName &> /dev/null
```
Comment (arora-aman) [inline bin/fk.sh]: One the mac pr is merged this would become `_fk_wrapper_open`

File: bin/fk.sh [hunk 2]
```diff
@@ -27,14 +27,20 @@ fclone()
   fcd $cdDir
 }
 
+remote-info()
+{
+  repoName=$(git config --get remote.origin.url | xargs basename | cut -d'.' -f1)
```
Comment (arora-aman) [inline bin/fk.sh]: This need to pull my fix to support https

Comment (arora-aman) [reply bin/fk.sh]: Also use local variables

**Review by arora-aman — DISMISSED** [review]

File: bin/fk.sh (hunk 2 above)
Comment (9at8) [reply bin/fk.sh]:
```suggestion
  repoName=$(basename $(git config --get remote.origin.url) .git)
```

**Comment by Azhng** [comment]
Review addressed @9at8  @arora-aman

**Pull request merged**